packages/trader/src/App/Components/Form/CompositeCalendar/composite-calendar-mobile.jsx

```diff
@@ -212,7 +229,7 @@ const CompositeCalendarMobile = React.memo(
                                     is_nativepicker={true}
                                     placeholder={localize('End date')}
                                     value={to}
-                                    max_date={today}
+                                    // max_date={today}
```

Comment:
Is this not required anymore?

Reply:
I made a new push to revert it back. This was for a test and forgot to uncomment it.